pkg/sql/randgen/schema.go

```diff
@@ -277,6 +277,14 @@ func generateInsertStmtVals(rng *rand.Rand, colTypes []*types.T, nullable []bool
 //
 // If numRowsInserted == 0, PopulateTableWithRandomData or RandDatum couldn't
 // handle this table's schema. Consider increasing numInserts or filing a bug.
+//
+// It's also worth noting that inserting data into a table with a virtually
+// computed column may cause a later select statement on the column to fail. E.g.
+// consider the following table: "CREATE TABLE math (foo INT4, baz INT4, bar AS
+// (foo + baz) VIRTUAL)"; and suppose that foo and baz had really large values
+// such that it causes bar's value to be out of INT4's range! A select * on the table
+// would then fail.
+//
```

Comment:
Is this redundant with the "- Out of range error for a computed INT2..." comment above? Seems like an example in which we want to throw the table away. Can we generate an allowlist of errors that we don't consider fatal and allow the test to continue?

Reply:
ah, yeah. this comment is redundant. As for the allowlist, I do that more informally above (via str matching), which I think is fine for now. It seems that how sql does error matching in its tests.